chia/plotting/manager.py

```diff
@@ -81,6 +103,32 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._lock.release()
 
+    def cache_path(self):
+        return Path(self.root_path / "plot_manager.dat")
+
+    def load_cache(self):
+        try:
+            serialized = self.cache_path().read_bytes()
+            self.log.info(f"Loaded {len(serialized)} bytes of cached data")
+            stored_cache: DiskCache = DiskCache.from_bytes(serialized)
+            if stored_cache.version < CURRENT_VERSION:
```

Comment:
this should probably be `!=`, since we can't load future format versions of this file..
or at least `>`

Reply:
I don't really get why you think `>` would be any better because the idea is to migrate from old cache format to new cache but i agree that `!=` makes more sense, changed.

Reply:
well, assuming we support `CURRENT_VERSION` as well as all previous formats, `>` would be the check to fail loading the file.

Reply:
oh, of course :see_no_evil: